### DISCUSSION TITLE
Browser cache issue. How to add chunkhash to the built static resource files?

### DISCUSSION THREAD

**Discussion by wanghehui1997** [coder/code-server#7029] [Q&A]
When I build the vscode, sometimes the browser doesn't update to the latest page resources but instead caches the previous  resources. How can I resolve this issue?

Is there any configuration available to add a timestamp or chunkhash to the static resources during the build process?

**Comment by code-asher** [comment]
No configuration as far as I am aware.  I think VS Code has some cache-busting issues, but they likely need to be fixed upstream.

The weird thing is, the resources have the commit in the URL so I am not sure how this happens, but I have definitely seen stale resources before.
